### PR TITLE
Split Checkout: Handle shipping/payment method description with no space

### LINKED ISSUE #6139: Shipping info on checkout screen on desktop does not span width of box and radio buttons are not acting as bullet points for the copy
Off the back of the mobile shopping UX work, where the edit cart and checkout screens had minimal changes made in this round, this is a small fix that will be super helpful when you have a large amount of content in the shipping info section (for an example of this look at the [AU Prom Coast Food Collective shipping info](https://openfoodnetwork.org.au/prom-coast-food-co-op/shop#/shop), they have order cycles open wed-mon)

## Expected Behavior
When going through checkout on a desktop device, the shipping options span the width of the shipping info section and if the copy does break onto multiple lines the copy on that second line is aligned with the top line of text (ie the radio button is like a bullet point)

## Actual Behaviour
The shipping options only span half the width, and the copy is aligned with the radio button (ie the radio button isn't like a bullet point) when there are multiple lines.

## Animated Gif/Screenshot
shipping methods layout crap

## Workaround
n/a

## Severity

bug-s1: a critical feature is broken: checkout, payments, signup, login
bug-s2: a non-critical feature is broken, no workaround
bug-s3: a feature is broken but there is a workaround
**bug-s4: it's annoying, but you can use it**
bug-s5: we can live with it, only a few users impacted

## Your Environment

* Version used: latest
* Browser name and version: Chrome latest
* Operating System and version (desktop or mobile): desktop Mac Mojave

### PR DESCRIPTION
**Pull request by jibees**
#### What? Why?

Closes #6139

I don't think this case is very relevant, but let's fix it!

#### What should we test?

- Create/Edit shipping method with long description _with_ space
- Create/Edit shipping method with long description _without_ space
- Create/Edit payment method with long description _with_ space
- Create/Edit payment method with long description _without_ space
- Check that everything's fine on both desktop and mobile device on every split checkout steps
- Check that everything's fine on order summary as well

Capture d’écran 2022-04-28 à 11 04 15
Capture d’écran 2022-04-28 à 11 04 39
Capture d’écran 2022-04-28 à 11 04 55
Capture d’écran 2022-04-28 à 11 21 03

#### Release notes

Changelog Category: User facing changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: app/webpacker/css/darkswarm/mixins.scss [hunk 1]
```diff
@@ -213,6 +213,5 @@
 }
 
 @mixin force-wrap {
-  white-space: pre-wrap;
```
Comment (mkllnk) [inline app/webpacker/css/darkswarm/mixins.scss]: You could have removed this detour before submitting the PR. :wink:

**Review by mkllnk — APPROVED** [review]
Nice. It's satisfying to pay some attention to detail, isn't it?

File: app/webpacker/css/darkswarm/mixins.scss (hunk 1 above)
Comment (jibees) [reply app/webpacker/css/darkswarm/mixins.scss]: Right. I've force-pushed.

**Comment by filipefurtad0** [comment]
Hey @jibees ,

On the pics below, following legend:

Left - editing the shipping method
Middle - Mobile iPhone 13
Right - Desktop Firefox/Ubuntu

For shipping methods, the description is now kept within the box boundaries:

**Before this PR**
image

**After this PR**
image

The same is verified for payment methods:

**After this PR**

image

The payment method name still goes out of screen, when if undescores are used instead spaces. This is probably very unlikely scenario, and we can see that the full with of the screen is used, when spaces are on the shipping/payment method **name** - so I think we're all good here:

image

image

Merging!

**Pull request merged**